analyses/rna-seq-expression-summary-stats/README.md

```diff
@@ -0,0 +1,181 @@
+
+- `results/cancer_group_all_cohort_mean_tpm.tsv`
+- `results/cancer_group_all_cohort_standard_deviation_tpm.tsv`
+- `results/cancer_group_all_cohort_mean_tpm_cancer_group_wise_z_scores.tsv`
+- `results/cancer_group_all_cohort_mean_tpm_cancer_group_wise_quantiles.tsv`
+- `results/cancer_group_all_cohort_mean_tpm_gene_wise_z_scores.tsv`
+
+The samples used in each `cancer_group` are listed in `results/cancer_group_all_cohort_sample_metadata.tsv`. The columns are 1) `cancer_group`, 2) the number of samples in the `cancer_group`, and 3) the comma separated list of `Kids_First_Biospecimen_ID`s of the samples in the `cancer_group`.
+
+#### Individual cohort summary statistics tables
+
+The following wide tables are generated using the methods described above. Rows are genes. Columns are `cancer_group_cohort`s, except that the first two columns are gene symbol and gene Ensembl ID. A `cancer_group_cohort` is a string that concatenates a `cancer_group` and a `cohort` by `___`, e.g. `Meningioma___PBTA`, `Neuroblastoma___GMKF`, and `Diffuse midline glioma___PBTA`. If one gene symbol matches to multiple Ensembl IDs, the value of the Ensembl ID column is a comma separated list of all Ensembl IDs, e.g. `ENSG00000206952.3,ENSG00000281910.1`. In `inpiut/ens_symbol.tsv`, `SNORA50A` is mapped to both `ENSG00000206952.3` and `ENSG00000281910.1`.
+
+- `results/cancer_group_individual_cohort_mean_tpm.tsv`
+- `results/cancer_group_individual_cohort_standard_deviation_tpm.tsv`
+- `results/cancer_group_individual_cohort_mean_tpm_cancer_group_wise_z_scores.tsv`
+- `results/cancer_group_individual_cohort_mean_tpm_cancer_group_wise_quantiles.tsv`
+- `results/cancer_group_individual_cohort_mean_tpm_gene_wise_z_scores.tsv`
+
+The samples used in each `cancer_group_cohort` are listed in `results/cancer_group_individual_cohort_sample_metadata.tsv`. The columns are 1) `cancer_group_cohort`, 2) the number of samples in the `cancer_group_cohort`, and 3) the comma separated list of `Kids_First_Biospecimen_ID`s of the samples in the `cancer_group_cohort`.
+
+#### Long summary statistics tables
+
+The following long tables are generated using the wide tables.
+
+- `long_n_tpm_mean_sd_quantile_group_wise_zscore.tsv.gz`
+- `long_n_tpm_mean_sd_quantile_gene_wise_zscore.tsv.gz`
+
+Each row of the long table is a tab-delimited record of the following columns, as suggested by @jharenza and @taylordm at <https://github.com/PediatricOpenTargets/OpenPedCan-analysis/pull/27#issuecomment-868035006>.
+
+- `gene_symbol`
+- `gene_id`
+- `cancer_group`
+- `cohort`
+- `tpm_mean`
+- `tpm_sd`
+- `tpm_mean_cancer_group_wise_zscore`/`tpm_mean_gene_wise_zscore`
+- `tpm_mean_cancer_group_wise_quantiles`
+- `n_samples`
+
+If the record is generated using all cohorts, the `cohort` column takes the value of `AllCohorts`.
```

Comment:
```suggestion
If the record is generated using all cohorts, the `cohort` column takes the value of `AllCohorts`.
```

Let's make this `PedOT` as with #45

Reply:
Will do. I will also update the README.md for recent changes, like JSON, EFO, MONDO, and so on.